docs/faq.rst

```diff
@@ -161,3 +161,40 @@ I am seeing an Exception error, what's wrong?
 
 Have a look at the :ref:`Troubleshooting section <troubleshooting>` to
 see what to do.
+
+If two users modify the same collection offline, how does that conflict get resolved?
+-------------------------------------------------------------------------------------
+
+It depends on which client you use. If you use kinto.js, for instance, you can set up
+three conflict resolution strategies:
+
+* SERVER_WINS: local changes are overridden by remote ones ;
+* CLIENT_WINS: remote changes are overriden by local one ;
+* MANUAL (default): handle them on your own.
+
+There is, of course, a convenient API to handle conflict one by one
+http://kintojs.readthedocs.io/en/latest/api/#resolving-conflicts-manually
+
+Would you recommend redis or postgres?
+--------------------------------------
+
+We recommend using postgres which is most focused lately, but you can still use redis.
+The app we've extracted Kinto from
+`was using redis <https://github.com/mozilla-services/readinglist/blob/4745ff1ae31b2e4dc22487fe6d96cc97a083c7fc/config/readinglist.ini#L4>`_.
+
+What about aggregation/reporting around data, is Kinto ready for that?
+----------------------------------------------------------------------
+
+No, and it will not. This is something that should be done on top of Kinto, with
+ElasticSearch for instance. In order to do this, you could listen to the events that
+Kinto triggers and send the data to your ElasticSearch cluster. There is a tutorial for
```

Comment:
We can add a link to the tutorial: http://kinto.readthedocs.io/en/latest/tutorials/write-plugin.html